site/resources/types.md

```diff
@@ -124,6 +124,14 @@ As smart contract accounts are essentially the contract codes that operate on th
 <<< @/../packages/core/src/account/smartContractAccount.ts#SmartContractAccount
 :::
 
+## `User`
+
+`User` is a type that defines the model for the user on a client. It includes identifiers for the user including `email`, `orgId`, `userId`, `userId`, `address` (the corresponding smart contract account address of type Address), and `credentialId`. You can use the [`useUser`](../react/useUser.md) react hook to look up a user.
```

Comment:
> `(the corresponding smart contract account address of type Address)`

What does this mean? User is the owner/signer EOA, how does this connect to smart contract account? AlchemySigner User can be used something other than the owner of smart contract account(s). A little bit confusing whether this address refers to the EOA address of the Signer/User that maps 1:1 with the user Id/email/credentialId or if this is something else.

Reply:
Would be great to see clarification on the relationship between User <> Alchemy Signer <> Smart Contract Account (Alchemy Account).

Reply:
I actually don't know the answer here. @moldy530 @avasisht23 can you help clarify?

Reply:
Pretty positive it is the signer address